test/controllers/api/v2/parameters_controller_test.rb

```diff
@@ -21,6 +21,17 @@ def setup
     assert_not_nil assigns(:parameters)
     parameters = ActiveSupport::JSON.decode(@response.body)
     assert_not_empty parameters
+    assert_equal parameters['total'], 1
+  end
+
+  test "should get all parameters of a specific domain" do
+    post :create, params: { :domain_id => domains(:mydomain).to_param, :parameter => valid_attrs }
```

Comment:
I don't think we should use the controller to create domains. Wouldn't it be better to use a factory to create items_per_page + 1 parameters and ensure they're all shown in the get?